src/BeamAdapter/component/engine/WireRestShape.h

```diff
@@ -121,14 +115,21 @@ class WireRestShape : public core::objectmodel::BaseObject
      void getCollisionSampling(Real &dx, const Real &x_curv) ;
      void getNumberOfCollisionSegment(Real &dx, unsigned int &numLines) ;
 
-     //TODO(dmarchal 2017-05-17) Please specify who and when it will be done either a time after wich
-     //we can remove the todo.
-     // todo => topological change !
-     void releaseWirePart();
-
      void rotateFrameForAlignX(const Quat &input, Vec3 &x, Quat &output);
 
 
+     /////////////////////////// Deprecated Methods  ////////////////////////////////////////// 
+
+     /// For coils: a part of the coil instrument can be brokenIn2  (by default the point of release is the end of the straight length)
+     Real getReleaseCurvAbs() const {
+         msg_warning() << "Releasing catheter or brokenIn2 mode is not anymore supported. Feature has been removed after release v23.06";
+         return 0.0;
+     }
+
+     void releaseWirePart() {
+         msg_warning() << "Releasing catheter or brokenIn2 mode is not anymore supported. Feature has been removed after release v23.06";
+     }
```

Comment:
[[deprecated]] feature should be used, to throw a warning at compile-time.

Reply:
done